### PR TITLE
mkisofs: add page

### PR DESCRIPTION
**Pull request by sbrl**
After accidentally committing to master and deleting it again, I bring you my latest page! This time it's a rather useful command that apparently can do all sorts of things, but I only use it for creating basic iso files for CDs.

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/mkisofs.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# mkisofs
+
+> Create ISO files from folders. Also aliased as `genisoimage`.
```
Comment (agnivade) [inline pages/linux/mkisofs.md]: Put the second sentence as a new line.

File: pages/linux/mkisofs.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Create an ISO from a folder:
+
+`mkisofs -o {{filename.iso}} {{path/to/source/folder}}`
```
Comment (agnivade) [inline pages/linux/mkisofs.md]: source/folder => source_folder. Same for the next example too.

File: pages/linux/mkisofs.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+- Set the disc label when creating an ISO:
+
+`mkisofs -o {{filename.iso}} {{path/to/source/folder}}`
```
Comment (agnivade) [inline pages/linux/mkisofs.md]: This example is same as the previous one. Perhaps you wanted to add some flag in this one.

Comment (sbrl) [reply pages/linux/mkisofs.md]: Oops! Good catch. Updating now.

**Comment by sbrl** [comment]
Thanks, @agnivade! Is that better?

**Pull request merged**